### DISCUSSION TITLE
f复现代码后，帧率与文章中写的差距太大了，我的估计就10帧左右。有大佬遇到过类似的问题么？求解疑

### DISCUSSION THREAD

**Discussion by baigang666** [STVIR/pysot#143] [Reproduce] [answered]

**Comment by charlesyann** [comment]
trainning代码到现在还没有复现出作者paper的精度，你是怎么实现的呢？求指导

**Reply by cunjif** [reply]
请问你用了全部的数据集吗？gpu是几个？

**Comment by ZhiyuanChen** [comment]
> _No description provided._

你的机器配置是？

**Comment by ZhiyuanChen** [comment]
> trainning代码到现在还没有复现出作者paper的精度，你是怎么实现的呢？求指导

新的问题请开新的issue

**Comment by ZhiyuanChen** [comment, marked as answer]
由于本issue没有得到更进一步的提问，我现在将其关闭。
如有新的问题，请重新打开它或者创建一个新的issue。
